_observing-your-data/query-insights/top-n-queries.md

```diff
@@ -96,6 +94,9 @@ GET /_insights/top_queries?type=memory
 ```
 {% include copy-curl.html %}
 
+If your query returns no results, ensure that top N is enabled for the target metric type and that search requests were made within the current [time window](#configuring-the-window-size).
```

Comment:
Do we want to mention that the sorting of the results will be based on the metric type provided?